plugins/pulp_python/plugins/models.py

```diff
@@ -41,24 +41,33 @@ class Package(FileContentUnit):
     (author, summary). By keeping the scope of the metadata as small as is reasonable, we have less
     complexity and reduced risk of innacuracy.
 
+    :ivar author: primary author of the package
+    :type author: basestring
+    :ivar filename: name of the file containing the package and metadata
+    :type filename: basestring
+    :ivar md5_digest: md5 checksum provided by PyPI to ensure we got the correct bits
+    :type md5_digest: basestring
     :ivar name: name of the project, ex. scipy
     :type name: basestring
+    :ivar packagetype: format of python package, ex bdist_wheel, sdist
+    :type packagetype: basestring
+    :ivar url: url that the package can be downloaded from
+    :type url: basestring
     :ivar version: Contains the distribution's version number. This field must be in the format
                    specified in PEP 386.
     :type version: basestring
-    :ivar filename: name of the file containing the package and metadata
-    :type filename: basestring
-    :ivar author: primary author of the package
-    :type author: basesetring
     :ivar summary: one line summary of what the package does
     :type summary: basestring
     """
 
-    name = StringField(required=True)
-    version = StringField(required=True)
-    filename = StringField(required=True)
     author = StringField()
+    filename = StringField(required=True)
+    md5_digest = StringField()
```

Comment:
I'm not crazy about having the md5_digest and _checksum and _checksum_type.

While you're in here, can we change this to a dictionary called ``checksums`` or something where the keys are the checksum algorithm and the value is the hex digest? I think this is also happening in the pulp_rpm plugin so if you do this make sure they match. It's not as nice as unifying our file and checksum modeling, but it'll be consistent at least.